sdks/python/apache_beam/runners/common.py

```diff
@@ -565,9 +593,15 @@ def process_outputs(self, windowed_input_element, results):
     then dispatched to the appropriate indexed output.
     """
     if results is None:
+      # TODO(BEAM-3937): Remove if block after output counter released.
```

Comment:
Add to this TODO to verify performance is not adversely reflected.

Reply:
Updated https://issues.apache.org/jira/browse/BEAM-3937